docs/content/docs/development/scale-bucket.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Scale Bucket"
+weight: 5
+type: docs
+aliases:
+- /development/scale-bucket.html
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Scale Bucket
+
+Since the number of total buckets dramatically influences the performance, Table Store allows users to 
+tune bucket numbers by `ALTER TABLE` command and reorganize data layout by `INSERT OVERWRITE` 
+without recreating the table/partition. When executing overwrite jobs, the framework will automatically 
+scan the data with the bucket number recorded in manifest file and hash the record according to the current bucket numbers.
```

Comment:
`manifest file` is an internal concept.
Here we can simply say: reorganize the data according to the latest bucket number.
(number instead of numbers?)